api/database/factories/UserMocks.php

```diff
@@ -70,3 +70,16 @@
         'is_default' => $faker->boolean(),
     ];
 });
+
+$factory->define(\Spira\Rate\Model\Rating::class, function (\Faker\Generator $faker) {
+    return [
+        'rating_id' => $faker->uuid(),
+        'rating_value' => $faker->numberBetween(1, 10),
```

Comment:
rateable_id and rateable_type assigned through reletaion, no need to add it here